setup.cfg

```diff
@@ -112,4 +112,7 @@ python-tag = py38
 [coverage:run]
 omit =
     src/sentry/migrations/*
-source = .
+plugins = covdefaults
+
+[coverage:report]
+fail_under = 0
```

Comment:
```suggestion
fail_under = 0.01
```

covdefaults can't tell the difference between unset and default because `coverage` does not give it that information

Reply:
Oh, lame. Okay, fixed.